src/components/Dashboard/Dashboard.jsx

```diff
@@ -166,7 +166,7 @@ const Dashboard = () => {
 			</div>
 		</>
 	) : (
-		<OnType />
+		<OnType params={params} />
```

Comment:
Je moet de params niet doorgeven lijkt me. Je kan in de OnType componenten ook de params aanmaken met new URLSearchParams

Reply:
Klopt, maar ik zie dat deze al fixed zijn in de branch feat/dashboard-ontype.